src/Util/Strings.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @file src/Util/Strings.php
+ */
+
+namespace Friendica\Util;
+
+/**
+ * @brief This class contains methods to modify/transform strings.
+ */
+class Strings
+{
+    /**
+	 * escape text ($str) for XML transport
+	 * @param string $str
+	 * @return string Escaped text.
+	 */
+	public static function escape($str)
```

Comment:
Unlike @nupplaphil, I don't think it's a generic string escape, it is specifically for XML/HTML entities and the name should reflect that now that the class has been renamed away from `XML`. Please rename to `String::escapeXMLEntities`. Sorry I was late to bring my 2 cent before the rename.

Reply:
Are you sure that we really have to create such a function name monster? I don't have a problem for longer names when the function is only called a few times. But this function here is called very often.

Reply:
`XML::escape` was a great short descriptive name.

Reply:
Are you sure @MrPetovan ? It's a real low level escaping of strings. I think it would be useful for every kind of escaping of HTML text. Maybe we can name it `String::escape($html)` or similar to provide the information needed.

Reply:
OK, better back to `XML::escape` rather than the monster method name ;-)

Reply:
There are many ways of "escaping" a nondescript string:
- Adding backslashes before quotes
- URL encode
- JSON encode
- Escaping SQL punctuation
- XSS escaping

And I'd rather have explicit classes/method names for all of them.

Reply:
Sounds like I'm sending these back to XML::

Reply:
@zeroadam sorry for your additional work!

Reply:
Yeah, I'm really sorry as well, I had my head stuck in the new directory project, I didn't take any time to review PR this week-end.

Reply:
no worries!!